### PR TITLE
Fix math utils issue

### PR DESCRIPTION
**Pull request by kalwalt**
**⚠️ All PRs have to be done versus 'dev' branch, so be aware of that, or we'll close your issue ⚠️**

**What kind of change does this PR introduce?**

continue from PR #523

**Can it be referenced to an Issue? If so what is the issue # ?**
Yes see PR #523 and issue #515
**How can we test it?**

Test the Aframe and Threejs examples included in the repository, in the console it should print Aframe version 1.3.0 and no errors about MathUtils...
**Summary**

For more infos refer to the aforementioned issue and PR.
**Does this PR introduce a breaking change?**
No
**Please TEST your PR before proposing it. Specify here what device you have used for tests, version of OS and version of Browser**
Tested all the aframe examples.
**Other information**

### Todo list:
- [x] upgrade aframe version in all examples
- [x] upgrade Threejs npm package version to `150` and `three.min.js`, `three.js` included in the examples.
- [x] upgraded GLTFLoader.js to version r147. (to fix issue with nft example see https://github.com/AR-js-org/AR.js/pull/532/commits/0c70f716ca278e5c4971a2e58ab4d4b420502ba7)
- [x] upgrade three version for the examples
- [x] update github action scripts

### CODE REVIEW

**Comment by kalwalt** [comment]
https://github.com/AR-js-org/AR.js/pull/532/commits/f364a57e630312d9da70503e77240a61ee027e37 fixed issue https://github.com/AR-js-org/AR.js/issues/530

**Comment by kalwalt** [comment]
I upgraded three.js npm package to `150` and the three.js/examples/vendor/three.js/build/three.min.js and three.js/examples/vendor/three.js/build/three.js to the same version (the prreviuos was `132`). I didn't find any issue but if someone catch any issue let me know.

File: three.js/src/markers-area/arjs-markersareacontrols.js [hunk 1]
```diff
@@ -130,7 +130,7 @@ MarkersAreaControls.prototype._onSourceProcessed = function () {
     if (this.parameters.changeMatrixMode === "modelViewMatrix") {
       _this.object3d.matrix.copy(modelViewMatrix);
     } else if (this.parameters.changeMatrixMode === "cameraTransformMatrix") {
-      _this.object3d.matrix.getInverse(modelViewMatrix);
```
Comment (kalwalt) [inline three.js/src/markers-area/arjs-markersareacontrols.js:133]: same as above

File: three.js/src/markers-area/arjs-markersarealearning.js [hunk 2]
```diff
@@ -281,9 +281,9 @@ MarkersAreaLearning.prototype.toJSON = function () {
   };
 
   var originSubControls = this.subMarkersControls[0];
-  var originMatrixInverse = new THREE.Matrix4().getInverse(
```
Comment (kalwalt) [inline three.js/src/markers-area/arjs-markersarealearning.js:284]: idem

**Comment by nickw1** [comment]
Looks fine - happy to merge if all working as expected.

**Comment by kalwalt** [comment]
> Looks fine - happy to merge if all working as expected.

Yes, i will do i have done only tests on desktop, i will do on Mobile soon, if all goes fine i will merge it.

**Comment by kalwalt** [comment]
updated GLTFLoader.js to r147 and partially fixed an issue (caused by changes in three.js r150). We can't use the newer because it require to import AR.js  in a `<script type="module">`, but we don't have this feature yet. It's not a big deal to provide a jsm version of the lib but definetetly not the goal of this PR.
Note that in the three.js/nft example you will receive this warning:
`three.min.js:7 THREE.PropertyBinding: Trying to update node for track: mesh_0.morphTargetInfluences but it wasn't found`
I think it's not crucial we can skip for now.

**Comment by kalwalt** [comment]
@nickw1 i will make a new release as i think this is an awaited fix.

**Comment by nwcourses** [comment]
@kalwalt yes, I agree. (sorry for late reply but seen you have released anyway which is fine by me).

**Comment by nickw1** [comment]
@kalwalt I agree - sorry only just seen your update, I see you;ve released anyway which is great.

**Pull request merged**